BuildLoader.py

```diff
@@ -356,12 +355,13 @@ def update_fit_table (self):
             while ucode_offset < len(rom):
                 ucode_hdr = UCODE_HEADER.from_buffer(rom, ucode_offset)
                 if ucode_hdr.header_version == 1:
-                    if ucode_hdr.total_size:
-                        ucode_size = ucode_hdr.total_size
+                    u_code_images.append(ucode_offset)
+                    if (hasattr (self._board, "SLOT_SIZE")):
+                        ucode_offset += self._board.SLOT_SIZE
```

Comment:
The motivation for adding a slot size member to the board object is because I only want the "slotting" to occur only when explicitly indicated. I worry that this might introduce unintended side effects to legacy platforms like APL where this feature does not need to be introduced. Also, adding such a slot size member mirrors BIOS.
Adding a check of new vs. old slot size in the FW update code is non-trivial. This is because we would have to manually measure each uCode in the new image and the existing image. There already is a check on the sizes of the new uCode region vs. the existing uCode region, so other regions are safe. If the slot sizes are unaligned, it just simply won't boot.